content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/4-vault.md

```diff
@@ -0,0 +1,224 @@
+|Description (optional)   | Description of the app profile. | 
+|Tag (optional)           |  Assign tags to the app profile.|
+ 
+
+4. Select the **Vault** service and start the configuration.
+  
+
+
+5. Provide the following information to the wizard:
+  * **Name:** The application name.
+  * **PersistentVolumeClaim Size (GiB):** Select the volume as per the storage volume available in the cluster group and virtual clusters. Ensure you do not exceed the maximum storage size for your virtual cluster. 
+
+
+6. Save your changes.
+
+
+7. Deploy the app profile to a Palette Virtual Cluster. Use the [Deploy a Virtual Cluster](/clusters/palette-virtual-clusters/deploy-virtual-cluster#deployavirtualcluster) guide for additional guidance or check out the [Deploy an Application using Palette Dev Engine](/devx/apps/deploy-app) tutorial.
+
+
+## Validation
+
+You can validate the Vault instance deployed successfully by using the following steps. 
+
+1. Log in to [Palette](https://console.spectrocloud.com) and switch to **App Mode**.
+
+
+2. Navigate to the left **Main Menu** and select **Apps**.
+
+
+3. Select your application that contains Vault to view its details page.
+
+
+4. Ensure the **Status** is *Deployed* and that the service *Vault* has a green dot right next to it.
+
+
+5. Next, click on the **Virtual Cluster** link in the App details page.
+
+
+6. Download the **kubeconfig** file by clicking on the url. 
+
+
+7. Setup your local kubectl environment to use the kubeconfig file you downloaded. Review the [Access Cluster with CLI](/clusters/cluster-management/palette-webctl) guide for additional guidance.
+
+
+8. Export the following environment variables to prepare your environment to interact with Vault.
+
+    <br />
+
+    ```shell
+    export VAULT_ADDR=https://127.0.0.1:8200 
+    ```
+
+    <br />
+
+    ```shell
+    export VAULT_SKIP_VERIFY=true
+    ```
+
+9. The next step is to configure port forwarding between your local workstation and the pod hosting Vault. Use the following commands to configure the port forward.
+
+    <br />
+
+    ```shell
+    VAULT_NAMESPACE=$(kubectl get pods --selector app.kubernetes.io/name=vault --all-namespaces --output jsonpath='{.items[0].metadata.namespace}') && \
+    kubectl port-forward $(kubectl get pods --selector app.kubernetes.io/name=vault --all-namespaces --output jsonpath='{.items[0].metadata.name}') 8200:8200 --namespace $VAULT_NAMESPACE
+    ```
+
+    <br />
+
+    ```shell
+    kubectl port-forward $(kubectl get pods --selector app.kubernetes.io/name=vault --all-namespaces --output jsonpath='{.items[0].metadata.name}') 8200:8200 --namespace $VAULT_NAMESPACE
+    ```
+
+10. Open your browser and visit [https://localhost:8200/ui](https://localhost:8200/ui) to access the Vault UI. You will receive a warning due to the usage of a self-signed certificate but you can ignore this warning. 
```

Comment:
This link is broken.

Reply:
It will work if the user follows the steps :) otherwise it will not work.